ExportAs/ExportAs.py

```diff
@@ -54,14 +57,23 @@ def showContextMenuActionsForItem(self, itemID):
     subjectHierarchyNode = pluginHandlerSingleton.subjectHierarchyNode()
     self.exportAsAction.visible = True
     self.exportAsAction.enabled = False
+    self.exportTfmAsAction.visible = True
+    self.exportTfmAsAction.enabled = False
     self.menu.clear()
+    self.menu2.clear()
     associatedNode = subjectHierarchyNode.GetItemDataNode(itemID)
     if associatedNode is not None and associatedNode.IsA("vtkMRMLStorableNode"):
         self.exportAsAction.enabled = True
         writerExtensions = slicer.app.coreIOManager().fileWriterExtensions(associatedNode)
         for writerFormat in writerExtensions:
           a = self.menu.addAction(writerFormat)
-          a.connect("triggered()", lambda writerFormat=writerFormat : self.export(associatedNode, writerFormat))
+          a.connect("triggered()", lambda writerFormat=writerFormat : self.export(associatedNode, writerFormat)))  
+    if associatedNode is not None and associatedNode.IsA("vtkMRMLStorableNode") and associatedNode.GetTransformNodeID():
+        self.exportTfmAsAction.enabled = True
+        writerExtensions = slicer.app.coreIOManager().fileWriterExtensions(associatedNode)
+        for writerFormat in writerExtensions:
+          b = self.menu2.addAction(writerFormat)
+          b.connect("triggered()", lambda writerFormat=writerFormat : self.exportHardened(associatedNode, writerFormat))
```

Comment:
Use consistent terminology everywhere whenever possible.  So if the interface uses Transformed, then the related variables and code should also use Transformed.

Reply:
Right!